src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoPage.java

```diff
@@ -15,4 +16,14 @@
     public float getDefaultZoom() {
         return 1.0f;
     }
+
+    public Map<MultiblockAbility, List<String>> getAbilityTooltipMap() {
+        Map<MultiblockAbility, List<String>> abilityTooltipMap = new HashMap<>();
+        abilityTooltipMap.put(MultiblockAbility.EXPORT_ITEMS, Collections.singletonList("gregtech.multiblock.preview.any_hatch"));
```

Comment:
I wouldn't use Collections.singletonList here.

You would expect this code to work
````
@Override
public Map<MultiblockAbility, List<String>> getAbilityTooltipMap() {
    Map<MultiblockAbility, List<String>> result = super.getAbilityTooltipMap();
    List<String> exportItems = result.putIfAbsent(MultiBlockAbility.EXPORT_ITEMS, Lists.newArrayList());
    exportItems.add("my.tooltip");  // <--- Doesn't work?!
    return result;
}
````

It will actually throw an UnsupportedOperationException because that singleton list is immutable.